### PR TITLE
Bringing anuradha changes to resolve conflicts due to pr#653

### PR DESCRIPTION
**Pull request by boredabdel**
Bringing anuradha changes to resolve conflicts created in pr#653. I could not resolve them directly due to the branch protection in the repo.

### CODE REVIEW

File: tools/quota-monitoring-alerting/terraform/example.tfvars [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 20210 Google LLC
```
Comment (iht) [inline tools/quota-monitoring-alerting/terraform/example.tfvars:2]: 2021 :)

**Review by iht — APPROVED** [review]
LGTM

Only a minor comment, please fix copyright year in one of the headers.

File: tools/quota-monitoring-alerting/terraform/example.tfvars (hunk 1 above)
Comment (boredabdel) [reply tools/quota-monitoring-alerting/terraform/example.tfvars]: Fixed

**Pull request merged**